src/sentry/static/sentry/app/views/settings/components/forms/type.tsx

```diff
@@ -84,6 +84,9 @@ type BaseField = {
   meta?: string;
 
   selectionInfoFunction?: (props: any) => React.ReactNode;
+
+  stacked?: boolean;
+  flexibleControlStateSize?: boolean;
```

Comment:
Do we still need prop-types if we are using Typescript for a component? I think it might be redundant.

Reply:
@NisanthanNanthakumar Good question. We need prop types if we still are using them in JSX components. In any case, this is the TS definition.

Reply:
@scefali lol idk why I made the comment here instead on the SelectField component... But good to know!